source/bot/index.ts

```diff
@@ -11,6 +13,7 @@ import {i18n} from '../translation.js';
 import {menu} from './menu/index.js';
 import type {MyContext, Session} from './my-context.js';
 
+dotenv(); // Configure the .env file
```

Comment:
I think "load" is a better comment here than "configure" as nothing is really configured here?

```suggestion
dotenv(); // Load from .env file
```